### PR TITLE
Readme.md nits

### PR DESCRIPTION
**Pull request by Mistobaan**

### CODE REVIEW

**Comment by davecheney** [comment]
I think you meant 'nits' :)

File: README.md [hunk 1]
```diff
@@ -22,26 +22,26 @@ I gave a talk about `gb` and the rational for it's creation at GDG Berlin in Apr
...
 
 ## Incremental compilation
 
-By default `gb` always performs incremental compilation and caches the results in `$PROJECT/pkg/`. See the Flags section for options to alter this behaviour
+By default `gb` always performs incremental compilation and caches the results in `$PROJECT/pkg/`. See the Flags section for options to alter this behaviour.
 
 ## Flags
 
-The following flags are supported by `gb`. Note that these are flags to subcommands, so must come *after* the subcommand.
+The following flags are supported by `gb`. Note that these are flags to subcommands, so must they come *after* the subcommand.
```
Comment (davecheney) [inline README.md]: the previous version was correct

**Comment by davecheney** [comment]
Thanks, one small correction and I can merge this.

**Comment by Mistobaan** [comment]
fixed

**Comment by davecheney** [comment]
Thank you very much.

**Pull request merged**